Doofinder/Feed/Block/System/Config/CreateSearchEngine.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Doofinder\Feed\Block\System\Config;
+
+use Magento\Config\Block\System\Config\Form\Field;
+use Magento\Backend\Block\Template\Context;
+use Magento\Framework\Data\Form\Element\AbstractElement;
+
+
+class CreateSearchEngine extends Field
+{
+
+
+    protected $_template = 'Doofinder_Feed::System/Config/createSearchEngine.phtml';
+    public function __construct(Context $context, array $data = [],)
```

Comment:
I would remove the trailing comma in function params (implemented from PHP 7.3: https://wiki.php.net/rfc/trailing-comma-function-calls): 

```suggestion
    public function __construct(Context $context, array $data = [])
```